src/server/rdb_save.h

```diff
@@ -76,6 +77,9 @@ class RdbSaver {
     const StringVec search_indices;  // ft.create commands to re-create search indices
   };
 
+  // Dumps `obj` in DUMP command format. Uses default compression mode.
+  static std::string DumpObject(const CompactObj& obj);
```

Comment:
Did you add the implementation for this ? I dont see it

Reply:
Nope, it belongs to the next PR. Removed.